packages/mc-scripts/src/commands/build-vite.ts

```diff
@@ -77,6 +86,10 @@ async function run() {
       pluginSvgr(),
       pluginDynamicBaseAssetsGlobals(),
       pluginI18nMessageCompilation(),
+      process.env.ANALYZE_BUNDLE === 'true' &&
+        analyzer({ defaultSizes: 'parsed', openAnalyzer: true }),
+      process.env.ANALYZE_BUNDLE_TREE === 'true' &&
```

Comment:
This plugin creates a bundles dependency visualization:
![Screenshot 2024-10-08 at 18 32 05](https://github.com/user-attachments/assets/b7e4f21b-1bf6-4999-9569-e898501a45d8)